### ISSUE TITLE
One of my tracks doesn't scrobble, all other tracks are fine even tracks from the same album the problematic track is from

### ISSUE THREAD

**Issue by Artins90** [gix/foo_scrobble#10]
When I play the problematic track as "first track" Lastfm doesn't report any track being scrobbled.
If I switch to the problematic track from a working track as "second track" being played, Lastfm reports the previous track as if was still being scrobbled.

I have no idea what makes this track behave this way.

ID data:
Artist Name : LiSA
Track Title : Believe in myself
Album Title : Letters to U
Date : 2016
Genre : Rock, Pop
Composer : LiSA
Performer :  
Album Artist : LiSA
Track Number : 1/7
Total Tracks : 
Disc Number : 
Total Discs : 
Comment : JASRAC / Aniplex Inc.
<07F42305-3C75-529C-BA48-09435E88980D> :  
<45B1D925-1448-5784-B4DA-B89901050A13> : 10006001
<50DBF5A2-F864-5C17-BE00-C36DFD3DF7B4> :  
<8E90F26B-372A-5C8C-BB05-1EC0F36EE60C> : VVXX00269B14A
<93A74BEA-CE97-5571-A56A-C5084DBA9873> : JPE301100601
<ARTISTSORT> : LiSA
<BE242671-3D48-5AC8-B762-7D2DB4F584B8> : 11872907
<CATALOGID> : SVWJ-7760
<COPYRIGHT> : (P)2011 Aniplex Inc.
<COUNTRY> : Japan
<DISCOGS_RELEASE_ID> : 8313776
<FF8CA75F-2D68-52EB-85D6-1580486025A4> : 79718450
<GENRENUMBER> :  
<LYRICS> : [ti:Believe in myself] (...)
<MEDIATYPE> : Vinyl (LP, Mini-Album, Limited Edition)
<MOOD> :  
<MUSICBRAINZ_ALBUMID> : 50a943ce-2c4f-40b5-a249-cd796fbab9ee
<MUSICBRAINZ_ARTISTID> : 85d76093-9865-4605-97fa-8c910929d366
<MUSICBRAINZ_TRACKID> : 4a00386f-4c65-4b0f-b9e2-47248a193160
<PUBLISHER> : Aniplex
<STYLE> : J-pop, Pop Rock
<WWW> : https://www.discogs.com/LiSA-Letters-To-U/release/8313776

**Comment by gix** [comment]
What does foobar's log  (View -> Console) say? Most cases where a track is ignored print a message there with the reason.

**Comment by Artins90** [comment]
Hi,

Thank you for your reply, I didn't know foobar had a console. 
After looking at the console I found out what the issue was. Basically the track was not considered part of my library even though the file was in my library folder. 
I removed the song from the playlist and then I added it again, this fixed the issue somehow.
There was probably an issue with the media library/playlist feature of foobar itself.
